web/forms.py

```
@@ -1369,6 +1370,24 @@ def clean_deadline(self):
         return deadline
 
 
+class TeamGoalCompletionForm(forms.ModelForm):
+    class Meta:
+        model = TeamGoalMember
+        fields = ["completion_image", "completion_link", "completion_notes"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)